argocd/application-dev.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)